.eslintrc.json

```diff
@@ -1,28 +1,50 @@
 {
+  "root": true,
   "parser": "@babel/eslint-parser",
   "parserOptions": {
     "ecmaVersion": 2020,
     "ecmaFeatures": {
       "jsx": true
     },
-    "sourceType": "module"
+    "sourceType": "module",
+    "project": "./tsconfig.json"
   },
   "env": {
     "browser": true,
     "es6": true
   },
-  "extends": ["airbnb", "prettier", "prettier/react"],
-  "plugins": ["prettier", "react"],
+  "extends": [
+    "airbnb",
+    "prettier",
+    "prettier/react"
+  ],
+  "plugins": [
+    "prettier",
+    "react"
+  ],
   "settings": {
     "import/resolver": {
       "node": {
-        "extensions": [".js", ".jsx", ".ts", ".tsx"],
+        "extensions": [
+          ".js",
+          ".jsx",
+          ".ts",
+          ".tsx"
+        ],
         "paths": "src"
-      }
+      },
+      "typescript": {} // this loads <rootdir>/tsconfig.json to eslint
```

Comment:
This is a json file, unless interpreted as Javascript, comments will make this an invalid json.